mkdocs.yml

```diff
@@ -99,46 +99,39 @@ plugins:
           build: true
           nav_translations:
             Home: Accueil
-            Specification: Spécification
+            Get started: Premiers pas
+            Documentation: Documentation
             Current Version: Version actuelle
             Governance Process: Processus de gouvernance
             Data Quality: Qualité des données
-            Toolbox: Outils
-            Community Tools and Resources: Outils et ressources
-            Glossary: Glossaire
-            Learn: Apprendre
-            Guide: Guide
-            Participate: Participer
+            Data Policy: Politique en matière de données
+            FAQ: FAQ
+            Tools: Outils
+            Community: Communauté
         - locale: es
           name: Español
           build: true
           nav_translations:
             Home: Hogar
-            Specification: Especificación
+            Get started: Primeros pasos
```

Comment:
Shouldn't this be "Getting started" ?

Reply:
Both are valid :) I took inspiration from https://www.swift.org/